discord/ext/tasks/__init__.py

```diff
@@ -134,15 +140,49 @@ def __get__(self, obj, objtype):
         if obj is None:
             return self
 
-        copy = Loop(self.coro, seconds=self.seconds, hours=self.hours, minutes=self.minutes,
-                               count=self.count, reconnect=self.reconnect, loop=self.loop)
+        copy = Loop(self.coro, seconds=self._seconds, hours=self._hours, minutes=self._minutes,
+                               count=self._count, time=self._time, reconnect=self.reconnect, loop=self.loop)
```

Comment:
Might as well format this in a way to minify the chance of conflicts in the future.

```suggestion
        copy = Loop(
            self.coro, 
            seconds=self._seconds, 
            hours=self._hours, 
            minutes=self._minutes,
            count=self._count, 
            time=self._time, 
            reconnect=self.reconnect, 
            loop=self.loop,
        )
```